### PR TITLE
Map_Graphics Pt. 2 "Draw_Map_Graphics" Pullrequest

### PR DESCRIPTION
**Pull request by ukanyanwu**
With important structs for map_graphics pulled into the dev branch, the draw function for maps is ready to be added as well. Draw_map_graphics implements previously pulled structs from the maps module that allow developers to visualize the maps of the environment that players would go through in a chiventure game.

### CODE REVIEW

**Review by imadejski — APPROVED** [review]
Looks great!

File: graphics_sandbox/map_graphics/main.c [hunk 1]
```diff
@@ -0,0 +1,84 @@
+#include "raylib.h"
+#include "map_graphics.h"
+#include <stdlib.h>
+#include <stdio.h>
+
+
+/* See map_graphics.h for documentation */
```
Comment (philipchenaux) [inline graphics_sandbox/map_graphics/main.c:7]: The draw_map_graphics documentation in map_graphics.h is not added and is not complete (at least based on what I can see at the repo for the moment).

File: graphics_sandbox/map_graphics/main.c [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+        if (playerPosition.y > screenHeight - (screenHeight-MapTexture.height)/2) {
+            playerPosition.y = screenHeight - (screenHeight-MapTexture.height)/2;
+        }
+
+        // Draw
+        //----------------------------------------------------------------------------------
+        BeginDrawing();
+
+            ClearBackground(RAYWHITE);
```
Comment (philipchenaux) [inline graphics_sandbox/map_graphics/main.c:58]: Try to keep you indentation consistent throughout the code.

**Review by philipchenaux — CHANGES_REQUESTED** [review]
A few minor comments that need to be resolved.

File: graphics_sandbox/map_graphics/main.c (hunk 1 above)
Comment (ukanyanwu) [reply graphics_sandbox/map_graphics/main.c:7]: We've adjusted the header for our draw function to properly include the inputs and a SUCCESS when the function completes.

**Review by philipchenaux — APPROVED** [review]
Looks good to merge!

**Comment by philipchenaux** [comment]
Issue Score: Excellent

Comments:
Excellent job on this PR!

**Pull request merged**